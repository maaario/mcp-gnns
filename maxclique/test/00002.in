20 55
1 17
1 20
1 4
1 14
2 13
2 4
2 5
3 4
3 7
3 8
3 9
3 13
3 14
3 15
4 5
4 7
4 8
4 9
4 11
4 13
4 14
4 15
4 16
4 17
4 20
5 8
5 11
5 16
7 8
7 9
7 13
7 14
7 15
7 17
8 9
8 11
8 12
8 13
8 14
8 15
8 16
9 10
9 13
9 14
9 15
11 15
11 16
11 19
13 14
13 15
14 15
14 16
14 17
14 20
17 20