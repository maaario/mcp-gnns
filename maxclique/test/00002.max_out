8
2 3 6 7 8 12 13 14 
