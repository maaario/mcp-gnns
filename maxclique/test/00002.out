5 3 8 8 5 1 8 8 8 2 5 2 8 8 8 5 5 1 2 5 
